requirements_hpot.txt

```diff
@@ -0,0 +1,8 @@
+colorama
+SQLAlchemy
+SQLAlchemy-Utils
+cffi
+paramiko
+docker
+coverage
+numpy==1.16.4
```

Comment:
could we add pyyaml to these requirements

Reply:
I don't think we need colorama, I was using it at first but stopped using it after I changed to writing ML prediction results to an html report.